src/database/schema.sql

```diff
@@ -62,23 +62,5 @@ FOREIGN KEY (fk_session_id) REFERENCES sessions(id),
 FOREIGN KEY (fk_tutor_id) REFERENCES tutors(id)
 );
 
-INSERT INTO tutors (full_name, email, address, postal_code, phone_number, availability, fk_subject_id, fk_tutortype_id)
-VALUES 
-('John Doe', 'john.doe@example.com', '123 Tutor Lane', '12345', '07700900001', NULL, NULL, NULL),
-('Jane Smith', 'jane.smith@example.com', '456 Tutor St', '54321', '07700900002', NULL, NULL, NULL),
-('Alice Johnson', 'alice.johnson@example.com', '789 Tutor Blvd', '11223', '07700900003', NULL, NULL, NULL),
-('Robert Brown', 'robert.brown@example.com', '321 Tutor Rd', '33445', '07700900004', NULL, NULL, NULL),
-('Emily Davis', 'emily.davis@example.com', '654 Tutor Ave', '55667', '07700900005', NULL, NULL, NULL);
-
-INSERT INTO subjects (subject_name)
-VALUES 
-('Mathematics'),
-('Physics'),
-('Chemistry'),
-('Biology'),
-('English');
-
-
-
```

Comment:
I'm wondering if you could replicate the insert bug as it may be the way the db.ts file is setup. Could you leave this chunk of sql in? I want to see if there's an alternative to fixing the problem.